### ISSUE TITLE
For Keros, Date Picker control is showing , color contrast Error.

### ISSUE THREAD

**Issue by arunnagar15** [OfficeDev/office-ui-fabric-react#6179]
Bug Report
Package version(s) : (5.98.0)
Browser and OS versions : (browser- Chrome Version 68.0.3440.106 (Official Build) (64-bit) )
Priorities and help requested (not applicable if asking question):
Are you willing to submit a PR to fix? (No)

Requested priority: (High)

Products/sites affected: (Microsoft CRM UCI)

Describe the issue:
When application is run on chrome and tested using Keros, than Date picker , calendar icon is showing color contrast Error.

Actual behavior:
When application is tested on chrome , Keros shows color contrast issue on calendar icon of  Date picker control

Expected behavior:
Keros should not show such error for Date Picker control

**Comment by micahgodbolt** [comment]
@arunnagar15 can you specify what the error is? Is there a specific contrast value we should be hitting?

**Comment by Vitalius1** [comment]
@micahgodbolt it might be related to the use of `neutralTertiary` color and `neutralTertiaryAlt` on white background causing a contrast ratio issue like in the picture. We might need to get designers opinion on usage of those colors will usually fail Keros quick passes.
cc: @betrue-final-final 
![screen shot 2018-09-13 at 6 00 18 pm](https://user-images.githubusercontent.com/29514833/46386325-acf4d580-c675-11e8-90cf-7abba01442d6.png)

**Comment by micahgodbolt** [comment]
Yeah, i'll leave this to ben.
